lib/usdUfe/base/tokens.h

```diff
@@ -39,13 +39,16 @@ namespace USDUFE_NS_DEF {
     /* Stage received in the context of some router  */ \
     ((Stage, "stage"))                                  \
     ((EditTarget, "editTarget"))                        \
+    /* Metadata key path received in the context     */ \
+    ((KeyPath, "keyPath"))                              \
```

Comment:
Yes. More generally it provides the key for all dictionary-valued metadata. Currently there are only 2 cases:
- For `variantSelection`, it will simply provide the single-level`variantSet` name. 
- For USD `customData` edited through UFE metadata, `keyPath` will be composed of the UFE metadata group and name, eg "myorg:mycategory:foo."

Reply:
@pierrebai-adsk regarding the keyPath convention, are there other instances that we format the use case this way?

Alternatively we could have grouped metadata as we do in some of the UFE interfaces (setGroupMetadata) where the storage of the group is not concatenated with the rest of the path.

Reply:
It is the standard way to specify a composite metadata key path in USD.

Reply:
As the context seems more in the USD world - for example, it gives a prim instead of a scene item - I believe it's better to express the path the USD way,